src/chat/__tests__/narrowsSelectors-test.js

```diff
@@ -113,11 +113,11 @@ describe('getFirstMessageId', () => {
       narrows: Immutable.Map({
         [HOME_NARROW_STR]: [1, 2, 3],
       }),
-      messages: {
```

Comment:
```
tests: Use new eg.makeMessagesState where applicable.

Note that this actually changes the test data in a couple of spots --
where we were producing a malformed data structure!  Looks like a
copy-paste error... in which it's probably not a coincidence that the
old form here was pretty repetitive.
```

Does it change the test data? I'm not sure I follow; it looks like a pure refactor to me.

Reply:
Ah, it's subtle -- there's this (twice):
```diff
-      messages: {
-        ...eg.baseReduxState.messages,
-        [message1.id]: message1,
-        [message2.id]: message1,
-      },
+      messages: eg.makeMessagesState([message1, message2]),
```

Note the `message1` at `message2.id`.

I'll make the commit message clearer about that.

Reply:
Oh, yikes, yeah—that `message1`/`message2` misalignment sure looks fishy; thanks for fixing. I actually noticed this when working on my draft (not yet pushed) for storing `state.messages` as an `Immutable.Map` and made a note to return to it in case it was somehow done on purpose.